### PR TITLE
TBS[13]: OpenFaaS pre show notes

### PR DESCRIPTION
**Pull request by hasheddan**
Adds S3 bucket upload function guide with OpenFaaS + Crossplane.

TODO:
- Add UI func to display photos

Signed-off-by: hasheddan <georgedanielmangum@gmail.com>

### CODE REVIEW

File: episodes/13/assets/README.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+1. Install Crossplane using `arkade`
+
+```
+arkade install crossplane --helm3
+```
+
+1. Install `provider-aws`
+
+```
+kubectl apply -f provider-aws.yaml
```
Comment (alexellis) [inline episodes/13/assets/README.md:24]: Where do we get this file?

File: episodes/13/assets/README.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+9. Deploy function
+
+```
+faas-cli deploy -f tbs.yml
+```
+
+10. Post file
+
+```
+curl -X POST -F 'file=@test-image.jpg' http://127.0.0.1:8080/function/upload
```
Comment (alexellis) [inline episodes/13/assets/README.md:94]: Will there be a UI for this?

**Review by alexellis — CHANGES_REQUESTED** [review]
Couple of thoughts

**Comment by alexellis** [comment]
Are you going to add a list endpoint or a UI to view the uploads? Maybe they go to Slack?

**Comment by hasheddan** [comment]
@alexellis see PR body `TODO` for UI. Going to Slack would be fine as well

File: episodes/13/assets/README.md (hunk 2 above)
Comment (hasheddan) [reply episodes/13/assets/README.md:94]: Wasn't planning on it, but could add that relatively easily so might be a good idea.

**Comment by alexellis** [comment]
Cool 👍. I just looked at the files

File: episodes/13/assets/README.md (hunk 1 above)
Comment (hasheddan) [reply episodes/13/assets/README.md:24]: It is in the directory :+1:

**Comment by displague** [comment]
https://github.com/crossplane/tbs/blob/master/README.md should be updated with the latest link (we've missed a number of these so that could be a separate PR).

**Pull request merged**